.github/workflows/build.yaml

```diff
@@ -275,6 +275,8 @@ jobs:
     runs-on: ubuntu-20.04
     container:
       image: ghcr.io/${{ needs.workflow-setup.outputs.GITHUB_REPOSITORY_NAME }}/node-build:ubuntu-16-04
+    outputs:
+      INDY_NODE_PACKAGE_VERSION: ${{ steps.cache.outputs.pkgVer }}:
```

Comment:
This line is causing and error and stopping the workflow from being run.

![image](https://user-images.githubusercontent.com/10504350/149337024-651120ab-246c-49b5-8d1f-43a8fe891200.png)

Reply:
I'll fix it. The colon at the end of the line causes the error